packages/rancher-monitoring/rancher-monitoring/generated-changes/dependencies/rancherServiceMetrics/dependency.yaml

```diff
@@ -0,0 +1,2 @@
+workingDir: ""
+url: packages/rancher-monitoring/rancher-service-metrics
```

Comment:
can we move this to a different commit and have all the other stuff squashed into the first commit `Add rancher-service-metrics package`?

Reply:
Did the first have of this, can you specify which commits to squash together now that it's been updated?